cmd/call-endpoint/main.go

```diff
@@ -76,3 +90,21 @@ func main() {
 
 	cmd.MustHandleCMDRequestAndExitFull(args.Endpoint, request, nil, args.CommonOptions, printFunc)
 }
+
+func listAPIEndpoints() {
+	apiDescription, err := api.Describe(*api.GetRoutes())
+	if err != nil {
+		log.Fatal("Failed to describe API endpoints: '%w'.", err)
+	}
+
+	var endpoints []string
+	for endpoint, _ := range apiDescription.Endpoints {
+		endpoints = append(endpoints, endpoint)
+	}
+
+	sort.Strings(endpoints)
```

Comment:
I would use slices.Sort() instead.
It's not a big deal, but modern Go (since generics) just calls the slices version.
https://pkg.go.dev/sort#Strings